src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyboardHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyboardHandler);
+  }
+
+  keyboardHandler = (event: KeyboardEvent) => {
```

Comment:
![Screenshot 2023-03-02 at 18 35 15](https://user-images.githubusercontent.com/6526149/222492908-091690b5-c127-469b-9a24-90c70a122cf6.png)
```suggestion
  handleKeyUp = (event: KeyboardEvent) => {
```